stockticker/stockplot.py

```diff
@@ -0,0 +1,12 @@
+from bokeh.plotting import figure
+from bokeh.embed import components
+
+def plotting(prediction,df):
```

Comment:
space after comma

Reply:
also add a docstring for the function

Reply:
use # pylint: disable=C0103  (or the proper code) to get rid of warning about `df`.  I'm happy keeping that in since its a convention.